.buildbot.sh

```diff
@@ -2,5 +2,31 @@
 # vim: expandtab sts=0 sw=4 smarttab
 set -e
 
-echo "Test buildbot" 
+ROOT_DIR=$(pwd)/..
```

Comment:
Hmm, so I find this bit confusing. I *think* that `pwd` is the root of this repository, so why is the `ROOT_DIR` set to its parent?

Reply:
Looking below, I *think* I now see why you have `SRC_DIR=$1/build`. I think `ROOT_DIR` should not exist as a variable at all and... [I'll add extra comments below...]